sklearn/tree/_classes.py

```diff
@@ -853,19 +852,20 @@ def fit(self, X, y, sample_weight=None, check_input=True,
             ignored if they would result in any single class carrying a
             negative weight in either child node.
 
-        check_input : bool, (default=True)
+        check_input : bool, default=True
             Allow to bypass several input checking.
             Don't use this parameter unless you know what you do.
 
-        X_idx_sorted : array-like of shape (n_samples, n_features), optional
+        X_idx_sorted : array-like of shape (n_samples, n_features), \
+            default=None
```

Comment:
indent further for readability

```suggestion
            	default=None
```

Reply:
Nit: @alfaro96 This was not addressed yet.